Makefile

```diff
@@ -0,0 +1,29 @@
+#install kakarot
+
+install:
+	sh kakarotup
```

Comment:
Let me check, but I've received some bug report that sh kakarotup does not work, do you know if this is true? I'll check locally

Reply:
I guess we can address this in a later PR as you mentioned below!